imports/ui/app.jsx

```diff
@@ -33,8 +34,12 @@ class App extends Component {
   render () {
     const { userLoggedIn, errors, dispatch } = this.props
     const firstError = errors.length ? errors[0] : ''
+    var isIE = /*
+    @cc_on!@
+    */false || !!document.documentMode
     return (
       <div className='roboto'>
+        {isIE && <BrowserSupportMsg />}
         {userLoggedIn ? (
```

Comment:
This would still be rendered below the browser support message. It doesn't need to be